docs/adr/adr-0017-finding-accessible-colors-and-creating-design-tokens.md

```diff
@@ -0,0 +1,168 @@
+Having colors that can only be used with big texts (3:1 of contrast ratio) would have only made things more complicated.
+
+That is why we went for darker colors and avoided adding too much saturation so that new colors would not stand out too much from legacy colors still existing in the Console (menu background for instance).
+
+#### Tools
+
+To find these colors, we relied on the [Color Review](https://color.review/check/FEDC2A-5A3B5D) tool that we found thanks to [Stéphanie Walter's blog that lists many other great tools](https://stephaniewalter.design/blog/color-accessibility-tools-resources-to-design-inclusive-products/).
+
+#### Number of colors
+We chose to only keep 2 shades of our main colors (blue, green, orange and red):
+* the main color (to be used as a text color or a plain background that would make it stand out from the rest),
+* a very light shade of the color (to be used for background only, to make a text stand out but not too much).
+
+As for the grey color, we needed at least 10 shades since it is used in many different cases (2 shades for text, 2 shades for background, and shades for the differents states of the background like disabled, active, hovered, etc.).
+
+We know we may need to **add more shades or colors later on but we want to only select and add them when needed**.
+
+### Create Design Tokens
+
+The Design Token concept comes from the Salesforce team and has been defined by them as follows:
+> Design tokens are the visual design atoms of the design system — specifically, they are named entities that store visual design attributes. We use them in place of hard-coded values (such as hex values for color or pixel values for spacing) in order to maintain a scalable and consistent visual system for UI development.
+
+Source: [salesforce - Design Tokens](https://www.lightningdesignsystem.com/design-tokens/).
+
+Our work on Design Tokens is based on two articles:
+* [Design Tokens - What are they & how will they help you? by Lukas Oppermann](https://lukasoppermann.medium.com/design-tokens-what-are-they-how-will-they-help-you-b73f80f602ab)
+* [Naming Tokens in Design System by Nathan Curtis](https://medium.com/eightshapes-llc/naming-tokens-in-design-systems-9e86c7444676)
+
+These articles helped us grasp the concept of Design Tokens and find an architecture / strategy that would fit our needs.
+
+NOTE: Color Design Tokens are only the first step, see "What's next" section.
+
+#### No abstraction level
+
+Most of the examples we came accross insisted on building Design Tokens that were **Technology Agnostic and relied on tools to transform tokens into variables** that could be consumed with each specific technology (Web, Native mobile platforms, etc.).
+
+This made sense for very big projects distributed accross several different platforms but our project is a little bit different:
+* Our team is only made of developers and we don't use any specific design tool,
```

Comment:
Rephrased it based on your suggestion, is it better this way ? I am not sure the "no design tool" mention is a good idea ?